DiamondLightSource/python-artemis
# Tidy up run artemis - Removed deployment, improved help, tidier arguments

Fixes #603

### To test:
1. Confirm running ./run_artemis.sh --help combines help from the start-up script and artemis
2. Check other arguments still work as usual
3. Check other cleanup changes are suitable
